README.md

```diff
@@ -7,8 +7,9 @@
 ## Requirements
 
```

Comment:
- [ ] Please it will be nice to update your readme file with explicit steps to clone and run your app locally, as this will be of help to non technical users.